src/components/DotsNav.vue

```diff
@@ -4,7 +4,7 @@
       v-for="(section, index) in sections"
       :key="index"
       :class="[classes.dotSectionLi, {'active': section.active}]"
-      :aria-title="section.title"
+      :title="section.title"

```

Comment:
Conver to `aria-label` instead to avoid conflicts with the popover library while still providing accessible information

Reply:
I changed the attribute from `title` to `aria-label`